assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json

```diff
@@ -0,0 +1,2508 @@
+            "styles": ["italic", "regular"],
+            "variants": [
+              [0, 200],
+              [0, 300],
+              [0, 400],
+              [0, 600],
+              [0, 700],
+              [0, 900],
+              [1, 200],
+              [1, 300],
+              [1, 400],
+              [1, 600],
+              [1, 700]
+            ],
+            "metrics": {
+              "upm": 1000,
+              "asc": 1133,
+              "des": -388,
+              "tAsc": 1133,
+              "tDes": -388,
+              "tLGap": 0,
+              "wAsc": 1133,
+              "wDes": 388,
+              "xH": 500,
+              "capH": 692,
+              "yMin": -285,
+              "yMax": 1082,
+              "hAsc": 1133,
+              "hDes": -388,
+              "lGap": 0
+            }
+          },
+          "fontSize": 36,
+          "backgroundColor": {
+            "color": {
+              "r": 196,
+              "g": 196,
+              "b": 196
+            }
+          },
+          "lineHeight": 1.1,
+          "textAlign": "left",
+          "padding": {
+            "locked": true,
+            "horizontal": 0,
+            "vertical": 0
+          },
+          "type": "text",
+          "content": "<span style=\"font-weight: 900; color: rgba(0,0,0,0.15)\">Good design is aesthetic</span>",
+          "fontWeight": 400,
+          "width": 337,
+          "height": 94,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "basedOn": "616622fc-f2de-4f1a-b919-a37764baa73a",
+          "id": "6bb18ef8-aa4e-4ac0-8eb4-67505e38c150",
+          "x": 37,
+          "y": 373
+        }
+      ],
+      "backgroundColor": {
+        "color": {
+          "r": 255,
+          "g": 255,
+          "b": 255
+        }
+      },
+      "type": "page",
+      "id": "902b7a10-0a66-4f92-998d-6c2b0a92b2b5"
+    },
+    {
+      "backgroundOverlay": "none",
+      "elements": [
+        {
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "backgroundColor": {
+            "color": {
+              "r": 255,
+              "g": 255,
+              "b": 255
+            }
+          },
+          "x": 1,
+          "y": 1,
+          "width": 1,
+          "height": 1,
+          "mask": {
+            "type": "rectangle"
+          },
+          "isBackground": true,
+          "isDefaultBackground": true,
+          "type": "shape",
+          "id": "18a0059e-28e0-4144-baf1-6d1fc4d527ef"
+        },
+        {
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "backgroundTextMode": "NONE",
+          "font": {
+            "family": "Nunito Sans",
```

Comment:
And this is just "Nunito" not "Sans"

(figma)
<img width="294" alt="Screen Shot 2020-09-18 at 9 46 46 AM" src="https://user-images.githubusercontent.com/10720454/93623721-030ed080-f994-11ea-9067-70efcc559972.png">

(editor)
![Screen Shot 2020-09-18 at 9 46 36 AM](https://user-images.githubusercontent.com/10720454/93623735-0bffa200-f994-11ea-9926-b0407a10a990.png)